### ISSUE TITLE
"Accept all" suggestion

### ISSUE THREAD

**Issue by brodonn** [n3xah/issues#8]
When adding suggestions to your lists "Accept all" is misleading - I think it should either be called "Select all" or clicking accept all should automatically select and then confirm in one step

**Comment by brentbahry** [comment]
Yeah it really means "Mark all accepted". However, if you're performing that action, I don't see the need to also hit "Confirm" after, and we can probably just pop up the confirmation dialog like we do when you hit "Confirm". Thoughts @veronicamarie24?

**Comment by veronicamarie24** [comment]
@brentbahry I personally use ‘accept all’ frequently to simply mark them all accepted and then refine my selection

**Comment by brentbahry** [comment]
Maybe at least a rename
